tuf/tuf.go

```diff
@@ -394,18 +395,8 @@ func (tr *Repo) InitSnapshot() error {
 }
 
 // InitTimestamp initializes a timestamp based on the current snapshot
```

Comment:
We should also update this comment to not include the snapshot since we haven't updated the timestamp with the current state yet